### DISCUSSION TITLE
Could you check quick setup script with Cloudsmith mirror?

### DISCUSSION THREAD

**Discussion by naramdash** [rabbitmq/rabbitmq-website#1864] [General]
### Describe the bug

I want to install rabbitmq 3.13 with cloudsmith mirror but it is not working.

Maybe mirror server is broken.

### Reproduction steps

computer env: amd64 ubuntu 22.04 LTS

1. copy and paste quick setup script
2. rabbitmq-server 3.9 installed
3. after running script, apt update fails with many IGN

```
E: Failed to fetch https://ppa1.novemberain.com/rabbitmq/rabbitmq-server/deb/ubuntu/dists/jammy/main/binary-i386/Packages  404  Not Found [IP: 104.21.27.7 443]
```
![image](https://github.com/rabbitmq/rabbitmq-website/assets/24370799/71aed38e-3b5e-426e-afe0-15111acada5a)

### Expected behavior

After running quick setup script...

- `apt update` is working well
- rabbitmq-server 3.13 is installed

**Comment by naramdash** [comment]
After creating the issue, I noticed the above statements.

> The Cloudsmith repository has a monthly traffic quota that can be exhausted. For this reason, examples below use a Cloudsmith repository mirror. All packages in the mirror repository are signed using the same signing key.

But right now, the time is '2024-04-01T23:08:55.326Z', so... it is hard to think used all of quota.

**Reply by michaelklishin** [reply]
It has nothing to do with the quota, you can see an i386 architecture warning right in the very `apt` output you have shared.

While at it, you [won't be able to go from 3.9 straight to 3.13 in all likelyhood](https://www.rabbitmq.com/docs/upgrade) (without wiping node's data directory first, e.g. using `apt purge rabbitmq-server` or `rabbitmqctl reset`).

**Comment by michaelklishin** [comment]
You have `binary-i386` in the URL, which suggests you are trying to use the mirror on a 32-bit machine. The mirrors only provide 64-bit (x86-64) packages.

There are no plans to provide 32-bit packages.

**Reply by naramdash** [reply]
Looking at the i386 message, it certainly seems to be correct.

Thanks for the super quick reply!!!

However, I don't know why it starts looking for packages for 32bit, since I haven't done anything besides the quick setup script and my computer is a 64bit OS.

I'll have to look into this further.

**Comment by naramdash** [comment]
Quick start script working well on my ubuntu multipass vm 

Sorry to confusing you @michaelklishin

**Comment by naramdash** [comment]
If someone on AMD64 but see something about i386 packages...

I solved it by changing(add arch=amd64 for every line) the rabbitmq.list file in source.list.d to the following.


```
## Provides modern Erlang/OTP releases from a Cloudsmith mirror
##
deb [arch=amd64 signed-by=/usr/share/keyrings/rabbitmq.E495BB49CC4BBE5B.gpg] https://ppa1.novemberain.com/rabbitmq/rabbitmq-erlang/deb/ubuntu jammy main
deb-src [arch=amd64 signed-by=/usr/share/keyrings/rabbitmq.E495BB49CC4BBE5B.gpg] https://ppa1.novemberain.com/rabbitmq/rabbitmq-erlang/deb/ubuntu jammy  main

# another mirror for redundancy
deb [arch=amd64 signed-by=/usr/share/keyrings/rabbitmq.E495BB49CC4BBE5B.gpg] https://ppa2.novemberain.com/rabbitmq/rabbitmq-erlang/deb/ubuntu jammy main
deb-src [arch=amd64 signed-by=/usr/share/keyrings/rabbitmq.E495BB49CC4BBE5B.gpg] https://ppa2.novemberain.com/rabbitmq/rabbitmq-erlang/deb/ubuntu jammy main

## Provides RabbitMQ from a Cloudsmith mirror
##
deb [arch=amd64 signed-by=/usr/share/keyrings/rabbitmq.9F4587F226208342.gpg] https://ppa1.novemberain.com/rabbitmq/rabbitmq-server/deb/ubuntu jammy main
deb-src [arch=amd64 signed-by=/usr/share/keyrings/rabbitmq.9F4587F226208342.gpg] https://ppa1.novemberain.com/rabbitmq/rabbitmq-server/deb/ubuntu jammy main

# another mirror for redundancy
deb [arch=amd64 signed-by=/usr/share/keyrings/rabbitmq.9F4587F226208342.gpg] https://ppa2.novemberain.com/rabbitmq/rabbitmq-server/deb/ubuntu jammy main
deb-src [arch=amd64 signed-by=/usr/share/keyrings/rabbitmq.9F4587F226208342.gpg] https://ppa2.novemberain.com/rabbitmq/rabbitmq-server/deb/ubuntu jammy main
```
